<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArucoTestViewControls</class>
 <widget class="QWidget" name="ArucoTestViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>431</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please select an image!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSetupTracker">
     <property name="text">
      <string>Setup ArUcoTracker</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonStart">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonPerformImageProcessing">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Do Something</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSlice">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Slice it!</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item>
    <widget class="QCheckBox" name="boxSlicing">
     <property name="text">
      <string>Permanent Slicing</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnCalibrate">
     <property name="text">
      <string>Calibrate probe</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="boardDetectionLabel">
     <property name="text">
      <string>Board Detection </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSetImageGeo">
     <property name="text">
      <string>SetImageGeo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnWorldToImage">
     <property name="text">
      <string>WorldToImage</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSetImage">
     <property name="text">
      <string>SetReferenceImage</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
